Core/Vault.php

```diff
@@ -41,7 +42,8 @@ public static function getCards()
     {
         $oUser = Registry::getSession()->getUser();
         $sUserId = $oUser->getId();
-        $sAddressId = $oUser->getSelectedAddressId();
+        //FIXME get shipping address
```

Comment:
@tomazpu Address handling fixed in another PR to reduce the size of this one (see PR description):
https://github.com/wirecard/oxid-ee/pull/174/files